src/collections/blog/2022/2022-06-27-converting-docker-compose-files-and-kubernetes-manifests-to-meshery-pattern-files/index.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: "Converting Docker Compose files and Kubernetes Manifests to Meshery Pattern Files"
+date: 2022-06-27 01:15:00 -0600
+author: Ruturaj Mohite
+description: Converting Docker Compose files and Kubernetes Manifests to Meshery Pattern Files
+thumbnail: ./thumbnail.png
+type: Blog
+product: Meshery
+category: "Meshery"
+tags:
+ - Projects
+ - Meshery
+ - Patterns
+published: true
+featured: true
+---
+
+import { BlogWrapper } from "../../Blog.style.js";
+import Blockquote from "../../../../reusecore/Blockquote";
+
+import dashboardImg from "./dashboard.png";
+import applicationsPageImg from "./applications_page.png";
+import importAppImg from "./import_app.png";
+import clickDeployImg from "./click_deploy.png";
+import requestPayloadImg from "./request_payload.png";
+
+<BlogWrapper>
+
+<h3>Meshery Pattern Files</h3>
+<p>Meshery uses service mesh patterns that act as configurations of service meshes or applications to be deployed through Meshery. These configurations are written in a YAML format in a pattern file. Here's a small example pattern file to deploy a demo deployment of Istio through Meshery:
+
+```yaml
+name: IstioSM
+version: 1.0.1
+services:
+  istio:
+    type: IstioMesh
+    namespace: istio-system
+    settings:
+      version: 1.14
+      profile: demo
+```
+</p>
+<p>(Meshery pattern files are still under active development. Refer [the Service Mesh Patterns Design Spec](https://docs.google.com/document/d/1B2N78EdRiZF-yVo1-HY3syppwBBDumgMuYg6seD-AJ4) for more information)</p>
```

Comment:
You'll have to use the `<a>` tag here

Reply:
Whoops, copied this directly from the markdown